apps/browse/views.py

```diff
@@ -240,8 +240,7 @@ def __init__(self, request, base, category, key, default):
                                                     default)
 
     def filter_featured(self):
-        # Never fear this will get & with manual order and the base.
-        return Addon.objects.all()
+        return self.base_queryset.all()
 
     def order_featured(self, filter):
```

Comment:
While we are at it... As far as I can see this is the only method that has any sort of ordering applied. Wouldn't it be nice to remove the magic doing the ordering in the `filter()` method of the base class, and explicitly call this method in `filter_featured()` above ? Or is there something I am missing?